example_settings.ini

```diff
@@ -118,6 +118,31 @@ email_path =
 auth_file =
 worksheet_key =
 
+[header_column]
```

Comment:
Make sure you're not adding changes to a PR that aren't supposed to be part of it.  This file also appears in other PRs and isn't related to this one.